v2-apache/Dockerfile

```diff
@@ -72,50 +111,31 @@ COPY --from=build /usr/local/lib/libfuzzy.so.2.1.0                             /
 COPY --from=build /usr/local/bin/ssdeep                                        /usr/local/bin/ssdeep
 COPY --from=build /usr/share/TLS/server.key                                    /usr/local/apache2/conf/server.key
 COPY --from=build /usr/share/TLS/server.crt                                    /usr/local/apache2/conf/server.crt
+COPY src/etc/modsecurity.d/*.conf /etc/modsecurity.d/
+COPY v2-apache/conf/extra/*.conf /usr/local/apache2/conf/extra/
 
 RUN ln -s libfuzzy.so.2.1.0 /usr/local/lib/libfuzzy.so \
  && ln -s libfuzzy.so.2.1.0 /usr/local/lib/libfuzzy.so.2 \
  && ldconfig
-
-RUN sed -i -E 's|(ServerTokens) Full|\1 Prod|' /usr/local/apache2/conf/extra/httpd-default.conf \
- && sed -i -E 's|#(Include conf/extra/httpd-default.conf)|\1|' /usr/local/apache2/conf/httpd.conf \
- && sed -i -E 's|#(ServerName) www.example.com:80|\1 ${SERVERNAME}|' /usr/local/apache2/conf/httpd.conf \
+      
```

Comment:
Please, configure your IDE to automatically trim empty lines.

In [Codium](https://vscodium.com/),

* go to <kbd>File</kbd> ➜ <kbd>Preferences</kbd> ➜ <kbd>Settings</kbd>
* Search for "trim"
* Check all options found